lib/actions/queryObjects.js

```diff
@@ -0,0 +1,19 @@
+const { messages } = require('elasticio-node');
+const SugarCrm = require('../sugarcrm');
+
+exports.getModules = function getModules(cfg) {
+  const instance = new SugarCrm(cfg, this);
+  return instance.getModules(true);
+};
+
+exports.process = async function ProcessAction(msg, cfg) {
+  const instance = new SugarCrm(cfg, this);
+  const foundObjects = await instance.makeRequest(`${encodeURIComponent(cfg.module)}/filter`, 'POST', msg.body);
+  if (cfg.outputMethod === 'emitAll') {
+    this.emit('data', messages.newMessageWithBody(foundObjects.records));
+  } else { // "emitIndividually"
+    foundObjects.records.forEach((record) => {
```

Comment:
What if `foundObjects` doesn't have `records` child element?
Have you covered such case with unit test?

Reply:
I will be able to comment this on the next week.